src/styles/Book.module.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
+body {
+  font-family: 'Roboto Slab', serif;
+}
+
+.books_container {
+  padding-top: 2rem;
+  display: grid;
+  gap: 1rem;
+}
+
+.books-container,
+.books__divisor {
+  --margin-divisor: 2rem;
+
+  margin-block-start: var(--margin-divisor);
+}
+
+.books__divisor {
+  margin-block-end: var(--margin-divisor);
+  border: 1px solid #e8e8e8;
+}
+
+.book,
+.book__details {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
- [ ] I notice you repeated the rule `display: flex;` more than twice in your stylesheet. One of the best practices in CSS is to ensure that you don't make repeat style declarations in style sheets. I would advise creating a class that contains the declaration and using the class on JSX you desire to implement the style on. For example:

```css
.d-flex {
  display: flex;
}
```

Kindly implement this change on all declarations that are repeated more than twice in your style sheets. 👍 😎 You can read this [article](https://speckyboy.com/good-bad-css-practices/)

#### You could also use the CSS variable way to make your code clean. Read more on that in this [article](https://www.w3schools.com/css/css3_variables.asp)

Reply:
Changed it accordingly